src/flexmeasures_client/client.py

```diff
@@ -119,7 +125,7 @@ async def request(
                             if response.status < 300:
                                 break
                     except asyncio.TimeoutError:
-                        message = f"Client request timeout occurred while connecting to the API. Retrying in {self.polling_interval} seconds..."  # noqa: E501
+                        message = f"Client request timeout occurred while connecting to the API. Polling step: {polling_step}. Retrying in {self.polling_interval} seconds..."  # noqa: E501
                         logging.info(message)
```

Comment:
I'm not exactly sure what levels should be used. I think the polling is standard behavior so it's more an info message. But if info should be used as little as possible it will go to debug...

Reply:
changed this one to debug as well